README.md

```diff
@@ -129,7 +148,13 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+ERD stands for Entity Relationship Diagram, and is used to show how two
+entities are related.  In the context of databases, they are used to display
+how tables relate to one another.  In this example, a person table would have
+rows for individual people and the pets table rows for individual pets.  Since
+a person can have multiple pets, the tables would be connected by one-to many
+relationships between individual rows in the people table to one or more rows
+in the pets table.  
```

Comment:
:+1: although you can use an ERD to show how **more than two** entities are related.